snaps/learn/about-snaps/files.md

```diff
@@ -92,29 +95,52 @@ For example, running [`yarn mm-snap build`](../../reference/cli/subcommands.md#b
 
 ## Configuration file
 
-The Snap configuration file, `snap.config.ts`, should be placed in the project root directory.
-You can override the default values of the [Snaps CLI options](../../reference/cli/options.md) by specifying
-them in the `config` object of the configuration file.
+The Snap configuration file, `snap.config.js` or `snap.config.ts`, should be placed in the project
+root directory.
+You can override the default values of the
+[Snaps configuration options](../../reference/cli/options.md) by specifying them in the
+configuration file.
 For example:
 
-```ts
-import { resolve } from 'path';
-import type { SnapConfig } from '@metamask/snaps-cli';
+<Tabs>
+<TabItem value="JavaScript">
+
+```javascript title="snap.config.js"
+module.exports = {
+    bundler: "webpack",
```

Comment:
Since the latest version `bundler: 'webpack'` is the default, and we're planning to deprecate the `bundler` field at some point. I don't think we need to mention it in the docs.

Reply:
@Mrtenz Do you mean we should remove it from options reference entirely, or just here in the example config files? I went ahead and removed it from the example configs but kept it in the reference since it is not yet deprecated.

Reply:
Just removing it from the examples is fine!